solidity/remappings.txt

```diff
@@ -1,5 +1,8 @@
 @openzeppelin=../node_modules/@openzeppelin
+@openzeppelin/contracts-upgradeable/=../node_modules/@openzeppelin/contracts-upgradeable/
```

Comment:
doesnt the above line already resolve this?

Reply:
Unfortunately not. If I don't explicitly remap it, then it uses the upgradable contracts from inside the lib/eigenlayer-core sub module. Check the below line.